app/views/facility_notifications/index.html.haml

```diff
@@ -10,7 +10,22 @@
   = render "shared/transactions/top", tab: "notifications"
 
 - if @order_details.any?
-  = render "shared/transactions/table", order_details: @order_details
+  #table_billing
+    = form_tag url_for(controller: params[:controller], action: :send_notifications), id: "journals_create_form", name: "journals_create_form", method: :post, class: "form-horizontal" do
+      .row
+        .span12= yield :action_instructions
+      .row.table-actions
+        .span1.select_all_none= select_all_link
+        #zero_dollar_orders.span2.offset8
+          = label_tag :notify_zero_dollar_orders, class: "checkbox" do
+            = check_box_tag :notify_zero_dollar_orders
+            = text("send_zero_dollar_notifications", scope: "admin.transaction_search.actions")
+        .span1
+          .submit
+            = submit_tag text(:send_notifications, scope: "admin.transaction_search.actions"), class: ['btn', 'btn-primary']
```

Comment:
Can we be consistent in our use of symbols vs strings in the `text` method? It looks like we tend to favor strings. Also, let's use double quotes in the class (we're not always perfectly consistent right now, but that's the style we're aiming for).

Reply:
Good catch, this used to be `text(@order_detail_action...`, which is how I ended up with the symbol there.  I meant to clean it up but missed it.